### DISCUSSION TITLE
2-3 timestep simulation plot

### DISCUSSION THREAD

**Discussion by HettiePike** [IBAHCM/RPiR#195] [Archive] [answered]
Hi @IBAHCM/rpir-instructors ,

I'm having issues of the last part of practical 2-3 when I plot the timestep simulations. When timestep is set to 1 or 0.1 my plot works fine, but when plotting with timesteps of 3 or 5 I'm not getting the fluctuations or instability the way others in my group were. The plot below shows timestep<-3. Do you know what I might be doing wrong? 

Thanks :)
![timestep_3](https://user-images.githubusercontent.com/77628798/106531101-7cdbcc80-64e5-11eb-8eea-3eb787cc1918.png)

**Comment by skybluejamie** [comment]
Hi @HettiePike, it's going to look different depending on what your _recovery_ and _infection_ rates are. I'd recommend trying different values for them.

**Reply by HettiePike** [reply]
Hi, I just had them set as the default, ecoli.transmission <- 1/2 and ecoli.recovery <- 1/3 but I have just changed them and it just pulls the lines further apart rather than getting the instability shown by the zig-zag lines. I just thought if I can't find the fluctuation in and simulation becoming inaccurate, I cant write the paragraph that we need to include in our script.

This plot is from timestep <- 1 and ecoli.transmission <- 1/3 and ecoli.recovery <- 1/2, that can't be right?
![Rplot](https://user-images.githubusercontent.com/77628798/106628420-b1956580-6571-11eb-8793-061fde424280.png)

**Reply by richardreeve** [reply]
This looks right by the way - R0 is under 1, so the outbreak dies out.

**Comment by richardreeve** [comment, marked as answer]
Hi @HettiePike - sorry, I've just seen this. It's a bit hard to say without your repo. Can you tell me which one it is? 
You may find you need to increase the timestep to 10 or 20 to get instabilities depending on your exact settings.

**Reply by HettiePike** [reply]
Hi Richards, thanks I've managed to sort it now. I had missed the changed in my for function so I hadn't updated it to the scaled rates. Thanks for your help though!
